Add condensed Promotions for V2 (#66)
elasticpath/elasticpath-dev: docs/carts-orders/get-a-cart.md

## Response Example

The following response example shows condensed promotions for cart discounts within the included object. 

> When you create a cart discount rule promotion, an item discount rule promotion and a standard cart discount promotion, and apply these discounts to a cart with eligible items, then upon calling `GET {{baseUrl}}/carts/{{cartID}}?include=promotions` endpoint, the cart relationship object will have the promotion item. Aditionally, the `included` object will contain condensed promotions for the cart discounts.

Reviewer: ```suggestion
When you create a cart discount rule promotion, an item discount rule promotion and a standard cart discount promotion, and apply these discounts to a cart with eligible items, then upon calling `GET {{baseUrl}}/carts/{{cartID}}?include=promotions` endpoint, the cart relationship object has the promotion item. Aditionally, the `included` object contains condensed promotions for the cart discounts.
```
